src/common/paddle/index.ts

```diff
@@ -110,9 +107,15 @@ export const getProductPurchaseType = ({
     throw new Error('Core product id is not set');
   }
 
+  if (!remoteConfig.vars.plusOrganizationProductId) {
+    throw new Error('Plus organization product id is not set');
+  }
```

Comment:
Maybe we should move these checks in specific cases, so that one does not break because other one is missing?